service/resource/deployment/current.go

```diff
@@ -30,6 +31,14 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 
 			for _, item := range deploymentList.Items {
 				d := item
+
+				major, minor, patch, err := getVersionBundleVersionInfos(d.Labels)
+				if err != nil {
+					r.logger.Log("cluster", key.ClusterID(customObject), "warning", fmt.Sprintf("cannot to update current version bundle version metric for guest cluster: %#v", err))
```

Comment:
i think there's an extra `to` in this message

Reply:
Done.